tests/scripts/arm64_post_build.py

```diff
@@ -274,6 +275,10 @@ def main(args):
    shutil.copy2(lst_file, core_root)
    log("Copied %s to %s." % (lst_file, core_root))
 
+   # Update the url to reflect winarm32 tests
+   if arch == "arm":
+         g_test_url = g_test_url_winarm
```

Comment:
This indent is 9 spaces over instead of 6. Unfortunately python can be picky about indentation and it is better to keep it consistent throughout the file.